Source/WebCore/accessibility/ios/WebAccessibilityObjectWrapperIOS.mm

```diff
@@ -2107,19 +2107,19 @@ - (AXCoreObject*)detailParentForSummaryObject:(AccessibilityObject&)object
     // Use this to check if an object is the child of a summary object.
     // And return the summary's parent, which is the expandable details object.
     return Accessibility::findAncestor<AccessibilityObject>(object, true, [&] (const AccessibilityObject& object) {
-        const auto& tag = object.tagName();
-        if (tag == summaryTag)
+        const auto& elementName = object.elementName();
```

Comment:
This probably shouldn't be `const auto&` anymore since the return type is uint16_t rather than an `const AtomString&` like tagName was.